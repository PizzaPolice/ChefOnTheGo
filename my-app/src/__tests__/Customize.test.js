import React from 'react';
import ReactDOM from 'react-dom';
import { shallow, mount, render } from 'enzyme';
import Customize from '../components/customize.js'


it('Customize component renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Customize />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

// This test is shallow rendering our Customize component
describe('Customize Component', () => {
    it('Customize should render without throwing an error', () => {
        expect(shallow(<Customize />).exists(<form className='Customize'></form>)).toBe(true)
      })
    });