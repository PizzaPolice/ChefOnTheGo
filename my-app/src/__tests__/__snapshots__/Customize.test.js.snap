// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Customize Component should render initial layout 1`] = `
Array [
  <div
    className="outer-panel"
>
    <div
        className="panel-body"
    >
        <h2
            className="title"
        >
            Customize
        </h2>
        <div
            className="input-fields"
        >
            <div
                className="input-line"
            >
                Food Type:
                <select
                    name="foodtype"
                    onChange={[Function]}
                >
                    <option
                        value="chinese"
                    >
                        Chinese
                    </option>
                    <option
                        value="greek"
                    >
                        Greek
                    </option>
                    <option
                        value="italian"
                    >
                        Italian
                    </option>
                    <option
                        value="japanese"
                    >
                        Japanese
                    </option>
                    <option
                        value="mediterranean"
                    >
                        Mediterranean
                    </option>
                    <option
                        value="burgers"
                    >
                        Burgers
                    </option>
                    <option
                        value="dessert"
                    >
                        Dessert
                    </option>
                </select>
            </div>
            <div
                className="input-line"
            >
                Location:
                <input
                    name="location"
                    onChange={[Function]}
                    type="text"
                />
            </div>
            <div
                className="input-line"
            >
                Time:
                <input
                    name="time"
                    onChange={[Function]}
                    type="time"
                />
            </div>
            <div
                className="input-line"
            >
                Telephone:
                <input
                    name="telephone"
                    onChange={[Function]}
                    type="tel"
                />
            </div>
        </div>
    </div>
    <div
        className="panel-footer"
    >
        <Link
            replace={false}
            to="login"
        >
            <button
                className="btn back-btn"
            >
                Logout
            </button>
        </Link>
        <Link
            onClick={[Function]}
            replace={false}
            to="list_chefs"
        >
            <button
                className="btn next-btn"
            >
                Submit
            </button>
        </Link>
    </div>
</div>,
]
`;
